doc/source/comparison_with_sas.rst

```diff
@@ -0,0 +1,604 @@
+.. currentmodule:: pandas
+.. _compare_with_sas:
+
+Comparison with SAS
+********************
+For potential users coming from `SAS <https://en.wikipedia.org/wiki/SAS_(software)>`__
+this page is meant to demonstrate how different SAS operations would be
+performed in pandas.
+
+If you're new to pandas, you might want to first read through :ref:`10 Minutes to pandas<10min>`
+to familiarize yourself with the library.
+
+As is customary, we import pandas and numpy as follows:
+
+.. ipython:: python
+
+    import pandas as pd
+    import numpy as np
+
+
+.. note::
+
+   Throughout this tutorial, the pandas ``DataFrame`` will be displayed by calling
+   ``df.head()``, which displays the first N (default 5) rows of the ``DataFrame``.
+   This is often used in interactive work (e.g. `Jupyter notebook
+   <https://jupyter.org/>`_ or terminal) - the equivalent in SAS would be:
+
+   .. code-block:: none
+
+      proc print data=df(obs=5);
+      run;
+
+Data Structures
+---------------
+
+General Terminology Translation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. csv-table::
+    :header: "pandas", "SAS"
+    :widths: 20, 20
+
+    ``DataFrame``, data set
+    column, variable
+    row, observation
+    groupby, BY-group
+    ``NaN``, ``.``
+
+
+``DataFrame`` / ``Series``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``DataFrame`` in pandas is analogous to a SAS data set - a two-dimensional
+data source with labeled columns that can be of different types. As will be
+shown in this document, almost any operation that can be applied to a data set
+using SAS's ``DATA`` step, can also be accomplished in pandas.
+
+A ``Series`` is the data structure that represents one column of a
+``DataFrame``. SAS doesn't have a separate data structure for a single column,
+but in general, working with a ``Series`` is analogous to referencing a column
+in the ``DATA`` step.
+
+``Index``
+~~~~~~~~~
+
+Every ``DataFrame`` and ``Series`` has an ``Index`` - which are labels on the
+*rows* of the data. SAS does not have an exactly analogous concept. A data set's
+row are essentially unlabeled, other than an implicit integer index that can be
+accessed during the ``DATA`` step (``_N_``).
+
+In pandas, if no index is specified, an integer index is also used by default
+(first row = 0, second row = 1, and so on). While using a labeled ``Index`` or
+``MultiIndex`` can enable sophisticated analyses and is ultimately an important
+part of pandas to understand, for this comparison we will essentially ignore the
+``Index`` and just treat the ``DataFrame`` as a collection of columns. Please
+see the :ref:`indexing documentation<indexing>` for much more on how to use an
+``Index`` effectively.
+
+
+Data Input / Output
+-------------------
+
+Reading keyed-in values:
```

Comment:
Thanks, I think I'll change it to 'Constructing a DataFrame from Values` - SAS has very limited structures - the code is basically like `StringIO`, which is I was using the term 'reading'.